src/utils/vacation-days-utils.tsx

```diff
@@ -0,0 +1,51 @@
+import { Typography, Box } from "@mui/material";
+import { theme } from "../theme";
+import { Person } from "../generated/client";
+import strings from "../localization/strings";
+import { useState } from "react";
+import { personsAtom } from "../atoms/person";
+import { useAtom } from "jotai";
+
+/**
+ * Display persons vacation days
+ * @param Person timebank person
+ */
+export const renderVacationDays = (person: Person | undefined) => {
```

Comment:
If you remove the undefined type from the person parameter, you could avoid the necessity of checking for the person variable in the rest of the code in this function.